### ISSUE TITLE
I just wanted to ask if I need to update some offsets?

### ISSUE THREAD

**Issue by TrilloxR** [disrup7/Pasta-Fortnite-External#1]
By launching the .exe it shows up on the game when pressing Insert but the features like ESP and stuff like that are not working. But why??

**Comment by masterpaster-ritz** [comment]
its bc u need the drivers, all the offsets are updated. I am currently swapping out the drivers and ill let u know if that makes it work.

**Comment by masterpaster-ritz** [comment]
After sawing out the reading and writing it made me blue screen so ima stop before i break my pc lol. I would just paste all of the functions into evo or smth

**Comment by TryCnoob** [comment]
Do you have discord so we can stay in contact.?
masterpaster-ritz ***@***.***> schrieb am Di. 9. Nov. 2021 um
03:19:

**Comment by TryCnoob** [comment]
Do you have a discord Server or something to stay in contact?

**Comment by bigpaster12345** [comment]
@masterpaster-ritz the current driver in there is ioctls, however he was using the pasted flush file buffers, if you want i can help you fix the source on discord if you'd like

**Comment by bigpaster12345** [comment]
utilities.h shows the registry functions.

**Comment by neicun** [comment]
@bigpaster12345 who the fuck are you?
apparently rampagemod's topmost overlay been undetected on both eac & battleye, this source is mainly pasted too.. i doubt you know shit about ioctl, and no "ioctls" are not a driver? lmfao

**Comment by bigpaster12345** [comment]
> @bigpaster12345 who the fuck are you?
> apparently rampagemod's topmost overlay been undetected on both eac & battleye, this source is mainly pasted too.. i doubt you know shit about ioctl, and no "ioctls" are not a driver? lmfao

Obviously, IOCTLS are the comm method???? I never said it’s a driver alone. Second off we all fucking know it’s pasted, third of all this is using CreateWindow which will easily get you fucked on battleye and EAC (or well flagged for it)

who are you again?

**Comment by bigpaster12345** [comment]
> @bigpaster12345 who the fuck are you?
> apparently rampagemod's topmost overlay been undetected on both eac & battleye, this source is mainly pasted too.. i doubt you know shit about ioctl, and no "ioctls" are not a driver? lmfao

Okay, my wording was a little cursed it was like 10PM either way.

**Comment by masterpaster-ritz** [comment]
> @masterpaster-ritz the current driver in there is ioctls, however he was using the pasted flush file buffers, if you want i can help you fix the source on discord if you'd like

my discord is ritz#9009  ty

**Comment by neicun** [comment]
no you don't give a shit CreateWindow just don't set flags to topmost and don't set to same position as game..
you were born with a brain, use it.

**Comment by cambo32** [comment]
![VS](https://user-images.githubusercontent.com/102672909/160843727-f6a6e684-d87b-47d6-a20d-10063892ff29.png)
Please help I have a problem when i want to debug it says 

1>C:\Users\Source\Repos\Pasta-Fortnite-External\c1xx : fatal error C1083: Cannot open source file: 'driver.cpp': No such file or directory
1>  main.cpp
1>C:\Users\Source\Repos\Pasta-Fortnite-External\main.h(8,10): fatal error C1083: Cannot open include file: 'd3dx9.h': No such file or directory

what should i do ?

**Comment by cambo32** [comment]
> no you don't give a shit CreateWindow just don't set flags to topmost and don't set to same position as game.. you were born with a brain, use it.

![VS](https://user-images.githubusercontent.com/102672909/160843727-f6a6e684-d87b-47d6-a20d-10063892ff29.png)
Please help I have a problem when i want to debug it says 

1>C:\Users\Source\Repos\Pasta-Fortnite-External\c1xx : fatal error C1083: Cannot open source file: 'driver.cpp': No such file or directory
1>  main.cpp
1>C:\Users\Source\Repos\Pasta-Fortnite-External\main.h(8,10): fatal error C1083: Cannot open include file: 'd3dx9.h': No such file or directory

what should i do ?

**Comment by cambo32** [comment]
![VS2](https://user-images.githubusercontent.com/102672909/160844332-a60d8917-2ef3-46fa-9c74-7c98b4c9b030.png)

Please help I have a problem when i want to debug it says 

1>C:\Users\Source\Repos\Pasta-Fortnite-External\c1xx : fatal error C1083: Cannot open source file: 'driver.cpp': No such file or directory
1>  main.cpp
1>C:\Users\Source\Repos\Pasta-Fortnite-External\main.h(8,10): fatal error C1083: Cannot open include file: 'd3dx9.h': No such file or directory

what should i do ?

**Comment by cambo32** [comment]
> no you don't give a shit CreateWindow just don't set flags to topmost and don't set to same position as game.. you were born with a brain, use it.


![VS2](https://user-images.githubusercontent.com/102672909/160844332-a60d8917-2ef3-46fa-9c74-7c98b4c9b030.png)

Please help I have a problem when i want to debug it says 

1>C:\Users\Source\Repos\Pasta-Fortnite-External\c1xx : fatal error C1083: Cannot open source file: 'driver.cpp': No such file or directory
1>  main.cpp
1>C:\Users\Source\Repos\Pasta-Fortnite-External\main.h(8,10): fatal error C1083: Cannot open include file: 'd3dx9.h': No such file or directory

what should i do ?

**Comment by cambo32** [comment]
> > @bigpaster12345 who the fuck are you?
> > apparently rampagemod's topmost overlay been undetected on both eac & battleye, this source is mainly pasted too.. i doubt you know shit about ioctl, and no "ioctls" are not a driver? lmfao
> 
> Obviously, IOCTLS are the comm method???? I never said it’s a driver alone. Second off we all fucking know it’s pasted, third of all this is using CreateWindow which will easily get you fucked on battleye and EAC (or well flagged for it)
> 
> who are you again?


![VS2](https://user-images.githubusercontent.com/102672909/160844332-a60d8917-2ef3-46fa-9c74-7c98b4c9b030.png)

Please help I have a problem when i want to debug it says 

1>C:\Users\Source\Repos\Pasta-Fortnite-External\c1xx : fatal error C1083: Cannot open source file: 'driver.cpp': No such file or directory
1>  main.cpp
1>C:\Users\Source\Repos\Pasta-Fortnite-External\main.h(8,10): fatal error C1083: Cannot open include file: 'd3dx9.h': No such file or directory

what should i do ?

**Comment by cambo32** [comment]
> Do you have a discord Server or something to stay in contact?


![VS2](https://user-images.githubusercontent.com/102672909/160844332-a60d8917-2ef3-46fa-9c74-7c98b4c9b030.png)

Please help I have a problem when i want to debug it says 

1>C:\Users\Source\Repos\Pasta-Fortnite-External\c1xx : fatal error C1083: Cannot open source file: 'driver.cpp': No such file or directory
1>  main.cpp
1>C:\Users\Source\Repos\Pasta-Fortnite-External\main.h(8,10): fatal error C1083: Cannot open include file: 'd3dx9.h': No such file or directory

what should i do ?

**Comment by cambo32** [comment]
> By launching the .exe it shows up on the game when pressing Insert but the features like ESP and stuff like that are not working. But why??

whats ur discord
